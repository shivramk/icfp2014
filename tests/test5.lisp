(define (main) (+ 3 4))
