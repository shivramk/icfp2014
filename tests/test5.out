LDC 3
LDC 4
ADD
